arbitrum-docs/launch-orbit-chain/how-tos/orbit-chain-finality.md

```diff
@@ -0,0 +1,29 @@
+# Orbit Chain Confirmation and Finality
+
+## Child Chain Transactions
+
+Generally, transactions executed through the sequencer on Orbit chains [achieve finality](/tx-lifecycle) equivalent to their parent chain once the relevant transaction data has been [posted in a batch](/sequencer). This means that transactions on Orbit chains are considered final in minutes.
```

Comment:
```suggestion
Generally, transactions executed through the sequencer on Orbit chains [achieve finality](/tx-lifecycle.mdx) equivalent to their parent chain once the relevant transaction data has been [posted in a batch](/sequencer.dx). This means that transactions on Orbit chains are considered final in minutes.
```
Links should point to the file with the extension, this way Docusaurus can check if the link is broken or not.

Also, check this FAQ, in case you find it to be a better reference for the finality link: https://arbitrum-docs-git-orbit-finality-explanation-offchain-labs.vercel.app/for-devs/troubleshooting-building#how-many-block-numbers-must-we-wait-for-in-arbitrum-before-we-can-confidently-state-that-the-transaction-has-reached-finality

Reply:
(Please, also change the other links of the docs 🙏 )